depends/README.usage

```diff
@@ -0,0 +1,34 @@
+To build dependencies for the current arch+OS:
+    make
+To build for another arch/OS:
+  make HOST=host-platform-triplet && make HOST=host-platform-triplet
+  (For example: make HOST=i686-w64-mingw32 -j4)
+
+A prefix will be generated that's suitable for plugging into Bitcoin's
```

Comment:
Just substituting Armory for Bitcoin doesn't make sense, since Armory doesn't use autoconf, so I also modified the text a little.